### PR TITLE
Update build.zig for breaking change 

### PR DESCRIPTION
**Pull request by MFAshby**
https://github.com/ziglang/zig/pull/19597

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -134,5 +134,6 @@
 	path = tree-sitter-org
 	url = https://github.com/milisims/tree-sitter-org.git
 [submodule "tree-sitter-ziggy"]
+	# TODO switch back to kristoff-it when https://github.com/kristoff-it/ziggy/pull/29 is merged
 	path = tree-sitter-ziggy
-	url = https://github.com/kristoff-it/ziggy.git
+	url = https://github.com/MFAshby/ziggy
```
Comment (neurocyte) [inline .gitmodules]: We don't actually build ziggy anywhere here. There is no reason to change the ziggy upstream or the ziggy commit reference. Please remove it.

**Comment by neurocyte** [comment]
This PR should be merged into a new zig-0.13 branch.

File: tree-sitter-ziggy
Comment (neurocyte) [inline tree-sitter-ziggy:1]: I will update all submodules this evening. That will make this part obsolete.

**Review by neurocyte — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (MFAshby) [reply .gitmodules]: ah thanks, removed the submodule in https://github.com/neurocyte/tree-sitter/pull/1/commits/da8fc99e847bb5f196c1c5ed8c3b6c3a7fe2f1a9

**Comment by neurocyte** [comment]
merged to zig-0.13

**Pull request closed without merging**